packs/democratic_csi_nfs/README.md

```diff
@@ -0,0 +1,180 @@
+# `democratic-csi` CSI plugin
+
+This pack deploys two jobs that run the
+[`democratic-csi`](https://github.com/democratic-csi/democratic-csi)
+CSI plugin. The node plugin tasks will be run as a system job, and the
+controller tasks will be run as a service job.
+
+## Client Requirements
+
+This pack can only be run on Nomad clients that have enabled volumes and
+privileged mode for the Docker task driver. In addition, clients will
+need to have a source NFS volume mounted to any client host that runs
+the controller task.
+
+### Example NFS Server
```

Comment:
The `democratic-csi` plugin supports a _lot_ of different NFS and ZFS setups. I'm not sure if we want to try to support arbitrary configurations in this one pack. It'd be awfully nice, but the client requirements end up being a chore to document (and not feasible for us to test, where this one is pretty simple). Open to suggestion here though for sure.

Reply:
If we did want to support a broader range of configs, we'd want to have something more like a blob or object for the config file (ref https://github.com/hashicorp/nomad-pack-community-registry/pull/113#discussion_r848827033)

Reply:
I think there are ways we could handle both using the [sprig `fail`](http://masterminds.github.io/sprig/flow_control.html) function in the template.

Reply:
Because it's very hard to test without buying an appropriate NAS appliance, I'm going to leave this as-is for now and if someone comes along later and wants to improve upon it they can.

Reply:
My point in the earlier comment is that you could provide variables to ingest a shapely configuration for something that is easily testable, but allow for a blob in cases where we don't have the ability to know the configuration in advance. Then it would be up to the template to fail if one way or the other wasn't provided.